docs/tutorials/misc/300_otcp.ipynb

```diff
@@ -0,0 +1,780 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #34.        x=confidence_regions[idx, :, 1].flatten(),
</pre><p>Don't think <code>.flatten()</code> is needed. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>